app/admin/contributions.rb

```diff
@@ -52,7 +52,9 @@
     column :state do |contribution|
       Contribution.human_attribute_name("state/#{contribution.state}")
     end
-    column :pr_state
+    column :pr_state do |contribution|
+      Contribution.human_attribute_name("pr_state/#{contribution.pr_state}" )
```

Comment:
the human attribute name is for the attribute name, not the value, use the translation approach of enumerize

Reply:
Fixed, thanks!